x-pack/plugins/security_solution/server/integration_tests/telemetry.test.ts

```diff
@@ -298,7 +298,8 @@ describe('telemetry tasks', () => {
     });
   });
 
-  describe('endpoint-meta-telemetry', () => {
+  // FLAKY: https://github.com/elastic/kibana/issues/189192
+  describe.skip('endpoint-meta-telemetry', () => {
```

Comment:
@elastic/security-attention would be nice if someone can verify this. I did check myself but someone from the team giving a +1 on this would make me feel more confident about skipping.